recipes/puremagic/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "puremagic" %}
+{% set version = "1.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/cdgriffith/{{ name }}/archive/{{ version }}.tar.gz
```

Comment:
Can you get the tarball from PyPI instead?

The GitHub tarballs are not very reliable as there `sha256` changes when GitHub updates their Git installation.

Reply:
No, tarball from PyPI doesn't contain json file that is needed for correct behavior of the library. I've opened a PR that would fix tarball builds, until that we'll have to use github tarball

Reply:
Okay, in that case the license file can be removed.